web/components/editor/emoji/emoji-suggestion.ts

```diff
@@ -21,26 +26,41 @@ const rank = (c: string) => {
   return r < 0 ? 101 : r
 }
 
+const emojiArr = Object.entries(invertBy(shortcodes)).map(
```

Comment:
Kind of odd to store the file as a `shortcode: codepoint` map if you are immediately going to transform it into a `codepoint: [shortcodes]` list. Why not just store it how you want it?